src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
 // export function Person({ person }) {}
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName: pN } = person;
```

Comment:
it is not ok to use `pN` as name of variable, use more understandable names
just use partnerName without shorts